lib/pushmi_pullyu/swift_deposit.rb

```diff
@@ -0,0 +1,58 @@
+require 'openstack'
+
+class PushmiPullyu::SwiftDepositer
+
+  def initialize(connection: nil, container: nil)
+    raise 'conection can not be nil' if connection.nil?
+    raise 'container can not be nil' if container.nil?
+
+    user         = connection[:username]
```

Comment:
These variable assignments are not needed. Just plug them into the openstack connect on line 17. There never used again.